sstables/sstables.cc

```diff
@@ -1618,6 +1618,7 @@ sstable::write_scylla_metadata(const io_priority_class& pc, shard_id shard, ssta
     _components->scylla_metadata->data.set<scylla_metadata_type::Sharding>(std::move(sm));
     _components->scylla_metadata->data.set<scylla_metadata_type::Features>(std::move(features));
     _components->scylla_metadata->data.set<scylla_metadata_type::RunIdentifier>(std::move(identifier));
+    _components->scylla_metadata->data.set<scylla_metadata_type::CannotHavePartitionTombstones>(std::move(cannot_have_partition_tombstones));
```

Comment:
std::move for a bool seems like overkill. I would remove std::move as it's useless noise.

Reply:
without it the code doesn't compile ;)

Reply:
wh00t!!! What's the error?

Reply:
It was something with boost's variant's `operator=` being defined for rvalue-reference only

Reply:
that's because disk_set_... defines set as follow:     template <TagType Tag, typename T>
    void set(T&& value) {
        data[Tag] = disk_tagged_union_member<TagType, Tag, T>{std::forward<T>(value)};
    }

alternatively, we could perfect forward cannot_... to it rather than moving

Reply:
nah. std::forward is even more noise than std::move.